mne/source_estimate.py

```diff
@@ -2686,6 +2686,10 @@ def extract_label_time_course(stcs, labels, src, mode='mean_flip',
         degrees from one stc to the next.
     ``'max'``
         Max value within each label.
+    If encountering a ``ValueError`` due to mismatch between number of
+    source points in the subject source space and computed ``stc`` object set
+    ``src`` argument to ``fwd['src']`` to ensure the source space is
+    compatible between forward and inverse routines.
```

Comment:
The CI error seems legit, due to this warning in `source_estimate`:

```sh
/home/circleci/project/mne/source_estimate.py:docstring of mne.extract_label_time_course:67: WARNING: Definition list ends without a blank line; unexpected unindent.
```

Check your indentation or leed with an empty line.